### ISSUE TITLE
Color loss

### ISSUE THREAD

**Issue by Wickedness230** [wangruixing/DeepUPE#42]
Hi, thank you for your work. I wonder how to compute the color loss mentioned in your paper. I couldn't find any specific codes computing color loss.

**Comment by wangruixing** [comment]
You can use l2 normalization at first, and use tf.losses.cosine_distance to calculate the color loss.

**Comment by UtopiaHu** [comment]
https://www.zhihu.com/question/327139341/answer/760888764
